test/util/ChartUtils.spec.tsx

```diff
@@ -191,7 +192,7 @@ describe('getBandSizeOfAxis', () => {
 });
 
 describe('parseSpecifiedDomain', () => {
-  const domain = [20, 100];
+  const domain: [number, number] = [20, 100];
```

Comment:
This would be a SpecifiedDomain, right? 
```suggestion
  const domain: SpecifiedDomain = [20, 100];
```

Reply:
Quite possible - I still don't fully understand the Domain code.